data/authors.yml

```diff
@@ -2666,3 +2666,7 @@ phbailey:
 yquinn:
   name: Yanir Quinn
   email: yquinn@redhat.com  
+
+ishaby:
```

Comment:
This change is probably worth doing in a separate PR in case one of the other, simpler, RPs is ready before this one.